ml/learn.py

```diff
@@ -96,18 +92,20 @@ def stack_arrays(data, delimited_names, delimiter=','):
 
 def setup_argparse():
     parser = argparse.ArgumentParser(
-        description='Use sklearn to make a random forest regressor')
+        description='Use sklearn to make a regressor')
     parser.add_argument('-infile', help='.npz file with X and y data', default='results.npz')
```

Comment:
How are you currently generating this `.npz` file? Don't have one to test with at the moment

Reply:
Maybe we can write a function that generates an example `.npz` file to test on

Reply:
Though this may fall outside the scope of learn.py

Reply:
probably be useful to have a unit test with for it, so not sure it's out of scope

Reply:
in practice, i'm doing it with the translator.py in that same folder. adding tests to these components (as well as stringing them together) is a good idea. i have to brush up on writing unit tests, but need to get this in production pretty fast. i'll either do it or accrue the technical debt

Reply:
Happy to pay the debt once it's ready, can use tox/pytest the same as merlin does.